smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HTTPBindingProtocolGenerator.kt

```diff
@@ -228,95 +203,40 @@ abstract class HttpBindingProtocolGenerator : ProtocolGenerator {
     fun renderCodableExtension(
```

Comment:
Lots of encoder/decoder logic eliminated in this type since new codegen doesn't require coding keys or protocols.